pkgs/tools/misc/file/default.nix

```diff
@@ -7,31 +7,24 @@
 
 stdenv.mkDerivation rec {
   pname = "file";
-  version = "5.44";
+  version = "5.45";
 
   src = fetchurl {
     urls = [
       "https://astron.com/pub/file/${pname}-${version}.tar.gz"
       "https://distfiles.macports.org/file/${pname}-${version}.tar.gz"
     ];
-    sha256 = "sha256-N1HH+6jbyDHLjXzIr/IQNUWbjOUVXviwiAon0ChHXzs=";
+    hash = "sha256-/Jf1ECm7DiyfTjv/79r2ePDgOe6HK53lwAKm0Jx4TYI=";
   };
 
-  outputs = [ "out" "dev" "man" ];
-
   patches = [
-    # Backport fix to identification for pyzip files.
-    # Needed for strip-nondeterminism.
-    # https://salsa.debian.org/reproducible-builds/strip-nondeterminism/-/issues/20
-    ./pyzip.patch
-
-    # Backport fix for --uncompress always detecting contents as "empty"
-    (fetchurl {
-      url = "https://gitweb.gentoo.org/repo/gentoo.git/plain/sys-apps/file/files/file-5.44-decompress-empty.patch?h=dfc57da515a2aaf085bea68267cc727f1bfaa691";
-      hash = "sha256-fUzRQAlLWczBmR5iA1Gk66mHjP40MJcMdgCtm2+u1SQ=";
-    })
+    # Upstream patch to fix 32-bit tests.
+    # WIll be included in 5.46+ releases.
+    ./32-bit-time_t.patch
   ];
 
+  outputs = [ "out" "dev" "man" ];
```

Comment:
Nice!

Reply:
Not a new change I'm afraid. It's a bit confusing diff due to me plugging `patches` in a slightly different place. Reordered to make it more obvious.